common/src/main/java/com/wynntils/core/config/ConfigHolder.java

```diff
@@ -160,7 +160,9 @@ public boolean valueChanged() {
     }
 
     public void reset() {
-        setValue(defaultValue);
+        // deep copy because writeField set's the field to be our default value instance when resetting, making default
+        // value change with the field's actual value
+        setValue(ConfigManager.getGson().fromJson(ConfigManager.getGson().toJson(defaultValue), this.fieldType));
```

Comment:
Time to make a deepCopy util method with the json to/from hack?

Reply:
It is gson instance dependent. It would be a call for bugs later on. This should not be used this frequently except here.

Reply:
Why aren't other overlay settings such as alignment affected?